test/testinfra.py

```diff
@@ -47,6 +49,26 @@ def arg_parser():
     parser.set_defaults(verbosity=1)
     return parser
 
+def read_whitelist():
+    # Try to load the whitelist
+    whitelist = None
+    try:
+        wh = open(WHITELIST, "r")
+        whitelist = [x.strip() for x in wh.read().split("\n") if x.strip()]
+    except IOError as exc:
+        if exc.errno == errno.ENOENT:
+            pass
+        else:
+            raise
+
+    # The whitelist defaults to the current user
+    if whitelist is None:
```

Comment:
Also, there's no saying that the current username will match the github name anyway.